### PR TITLE
Adding myself as a new user

### PR DESCRIPTION
**Pull request by maxgds**
Following through the documentation at https://docs.publishing.service.gov.uk/manual/get-started.html and looking to add myself as a new user here.

### CODE REVIEW

File: modules/users/manifests/maxlehmann.pp [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Creates the maxlehmann user
+class users::maxlehmann {
+  govuk_user { 'maxlehmann':
+    fullname => Max Lehmann'',
```
Comment (cbaines) [inline modules/users/manifests/maxlehmann.pp]: You've got a couple of rouge quotes here `''`.

Comment (maxgds) [reply modules/users/manifests/maxlehmann.pp]: Thanks, now fixed.

**Comment by nickcolley** [comment]
@maxgds sorry I didn't introduce you to how we write commits (https://github.com/alphagov/styleguides/blob/master/git.md#example-1)

In this case we'd do something like 'Add Max Lehmann to manifest'.

We also tend to squash commits together if they're just to fix a small error in a previous commit.

**Comment by fofr** [comment]
Closing in favour of #6422

**Pull request closed without merging**